hatrac/model/storage/amazons3.py

```diff
@@ -56,7 +71,10 @@ def wrapper(*args):
             self = args[0]
             conn_tuple = None
             try:
-                conn_tuple = self._get_pooled_connection()
+                if deferred_conn_reuse:
```

Comment:
This change should be dropped once we extend webauthn2.util.PooledConnection.  We should always get a connection from the pool.  The only meaning of deferred_conn_reuse is whether this wrapper puts the connection back in the pool, or whether the called method will do it instead, e.g. at the end of a data generator sequence that is still reading from the connection after this wrapper returns.